src/errors/errorHandler.js

```diff
@@ -0,0 +1,11 @@
+function errorHandler(err, req, res, next) {
+  if (err.name === "UnauthorizedError") {
+    // jwt authentication error
+    return res.status(401).json({ message: "Invalid Token" });
```

Comment:
Why this error is not defined in common error file?

Reply:
Before i made middleware error handler for token validation , now i will move it in common file